liquibase/src/test/java/org/openmrs/liquibase/CoreDataTunerTest.java

```diff
@@ -149,7 +148,7 @@ public void shouldDetectThatSecondListDoesNotContainFirstList() {
 			List<String> first = Arrays.asList("one", "six");
 			List<String> second = Arrays.asList("one", "two", "three", "four", "five");
 			
-			List<String> actual = mergeLists(first, second);
```

Comment:
is the actual list necessary?

Reply:
I don't think it is but it's not related to your story. Please only change things related to your story. Otherwise you'll end up changing tons of things as they is lots to work on :)
Also please check my first review summary. I think I said that we should exclude the liquibase subproject for now. So we get to merge some changes more quickly by having less files to review.